infra/charts/feast/charts/feast-core/values.yaml

```diff
@@ -64,9 +64,9 @@ application.yaml:
       options: {}
       metrics:
         enabled: false
-        type: prometheus
+        type: statsd
         host: localhost
-        port: 9091
+        port: 8125
```

Comment:
I believe the container uses 9125. You sure this is right?

Reply:
my bad. I thought it was using the default statsd port